src/main/java/org/broadinstitute/hellbender/engine/spark/datasources/ReadsSparkSource.java

```diff
@@ -238,7 +238,17 @@ private void setHadoopBAMConfigurationProperties(final String inputName, final S
                     throw new UserException("A 2bit file cannot be used as a CRAM file reference");
                 }
                 else { // Hadoop-BAM requires the reference to be a URI, including scheme
-                    String referenceURI = null ==  new Path(referenceName).toUri().getScheme() ?
+                    Path refPath = new Path(referenceName);
+                    try {
+                        FileSystem fs = refPath.getFileSystem(ctx.hadoopConfiguration());
+                        if (!fs.exists(refPath)) {
```

Comment:
Extract to a utility method that can check whether a `Path` exists given a `SparkContext`